seminars/practice_3_lists.py

```diff
@@ -1,191 +0,0 @@
-"""
-Programming 2023
-Seminar 3
```

Comment:
Please, revert this file! Instructions: https://github.com/fipl-hse/2023-2-level-labs/blob/main/docs/public/starting_guide_ru.md#continue-working